city/crud.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import HTTPException
+
+from sqlalchemy import select, insert, delete
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from city import models, schemas
```

Comment:
The import statement 'from city.crud import get_all_cities' seems to be redundant if this function is defined in the same file. Consider removing this import to avoid confusion.